packages/web-extension/src/messaging/runContentScriptMessageProxy.ts

```diff
@@ -12,27 +12,39 @@ export interface MessagingApi {
   callApiMethod(request: MethodRequest): Promise<unknown>;
 }
 
+interface ChannelMessageEvent {
+  channelName: string;
```

Comment:
Let's rename this everywhere to not confuse with Messenger's channelName
```suggestion
  apiName: ApiName;
```

Reply:
I assume we need to create `ApiName` as an opaque type. Are there any constraints on the type?

Reply:
No constraints, just an opaque string. FIY in xsy-wallet the type is called `Tagged` and we use the type util from type-fest instead of SDK's `OpaqueString`. It's the same thing.

Reply:
Since OpaqueString is already widely used in the SDK, I stuck with it on 8b69fdc